### ISSUE TITLE
SerialNumber is always null on Android

### ISSUE THREAD

**Issue by alebergmann** [woodemi/quick.flutter#46]
SerialNumber is always null on Android (using android 11). 
For targets Android 10 or higher, the app cannot read the serial number until the user has granted the app permission to access the USB device or accessory. 
QuickUsb is not requesting permission.

**Comment by Sunbreak** [comment]
https://github.com/woodemi/quick.flutter/tree/master/packages/quick_usb#request-permission

**Comment by alebergmann** [comment]
Yeah but 'hasPermission' returns true and the SerialNumber is null.
